### PR TITLE
EDU-2605: Adds missing metric plus caveat

### PR DESCRIPTION
**Pull request by fairlydurable**
Adds activity_succeed_endtoend_latency link and coverage

Source of truth: Antonio MP @antmendoza

### CODE REVIEW

File: docs/references/sdk-metrics.mdx [hunk 1]
```diff
@@ -163,6 +164,15 @@ from the queue.
 - Available in: Core, Go, Java
 - Tags: `namespace`, `task_queue`
 
+### activity_succeed_endtoend_latency
+
+Total latency of successfully finished Activity Executions from the time they are scheduled to the time they are completed.
+This metric is not recorded for async Activity completion.
+
+- Type: Histogram
+- Available in: Go, Java
```
Comment (antmendoza) [inline docs/references/sdk-metrics.mdx:173]: I am not sure if Go reports this metric (and Core). Java does. 
I can figure it out if you want.

**Review by antmendoza — COMMENTED** [review]
Thanks @fairlydurable

File: docs/references/sdk-metrics.mdx (hunk 1 above)
Comment (fairlydurable) [reply docs/references/sdk-metrics.mdx:173]: @antmendoza Yes please do! This is from my search yesterday:

image

**Review by napcs — APPROVED** [review]
LGTM

**Pull request merged**